src/Microsoft.PowerShell.Commands.Utility/commands/utility/trace/MshHostTraceListener.cs

```diff
@@ -62,35 +62,8 @@ internal PSHostTraceListener(PSCmdlet cmdlet)
         [SecurityPermission(SecurityAction.LinkDemand)]
         protected override void Dispose(bool disposing)
         {
-#if CORECLR
             base.Dispose(disposing);
-#else
-            try
-            {
-                if (disposing)
-                {
-                    this.Close();
-                }
-            }
-            finally
-            {
-                base.Dispose(disposing);
-            }
-#endif
-        }
-
-#if !CORECLR
-        /// <summary>
-        /// Closes the dialog and then calls the base class Close
-        /// </summary>
-        [SecurityPermission(SecurityAction.LinkDemand)]
-        public override void Close()
-        {
-            // Call the base class close
-
-            base.Close();
```

Comment:
How the method is implemented in the base class is implementation details. Think about `base.Dispose(disposing);`, it currently also does nothing in .NET Core : https://github.com/dotnet/corefx/blob/7da531794fe1f77668b0705502cd0658eba75889/src/System.Diagnostics.TraceSource/src/System/Diagnostics/TraceListener.cs#L79, but does that mean we can avoid calling it as well?

But I think we can remove the method `public override void Close()`. It doesn't nothing other than calling `base.Close()`, which is the same as directly calling `Close()` from the base type.